ballerina/types.bal

```diff
@@ -23,3 +23,48 @@ public type FieldMetadata record {|
 |};
 
 public type FilterQuery distinct sql:ParameterizedQuery;
+
+# Defines the entities of the table.
+#
+# + key - The primary key of the table
+# + unique - The unique index of the tavle
+# + tableName - The table name
+public type EntityConfig record {|
+    string[] key;
+    string[][] unique?;
```

Comment:
Shall we change the variable name `unique` -> `uniqueIndex`?

Reply:
This name is defined in the design doc. 

@daneshk Please suggest your suggestion. And, don't we need to use plural terms because the key and Unique allow taking multiple values?

Reply:
We need to revisit the field names. May be `primaryKey`, 'uniqueKeys'?
what is the correct term: is it `uniqueKeys` or `uniqueIndexes`?

Reply:
Change the `unique` to `uniqueConstraints` in [2e3257e](https://github.com/ballerina-platform/module-ballerina-persist/pull/2/commits/2e3257e9f4bdaf185c56899e09393fc242c99118)